tests/serverpod_test_server/lib/src/custom_classes.dart

```diff
@@ -14,7 +14,7 @@ class CustomClass implements SerializableModel {
   @override
   String toJson() => value;
 
-  static CustomClass fromJson(dynamic data) {
+  factory CustomClass.fromJson(dynamic data) {
```

Comment:
Suggestion: For the class - Update the import in this class to `serverpod` instead of the `serverpod_client`.